File: 02_Lenguajes-de-programación/01_Python/05_bucles.py
# BUCLES

# 1. Bucle For.
animales1 = ['perro', 'tigre', 'elefante']
for animal in animales1:
    print(animal) # Imprimirá el arreglo de los animales


def multiplicar(primero, segundo):
    print(primero * segundo)

numbers = [23, 564, 789, 12, 65, 10]

for number in numbers:
    multiplicar(number, 2) # Imprimirá los numeros pero multiplicados por el segundo valor.

# 2. Bucle While
entero2 = 100
emergencia = 911

while entero2 < emergencia:
    print(entero2)
    entero2 += 1

File: 02_Lenguajes-de-programación/03_C#/05_bucles.cs
using System;

class Program {
  public static void Main (string[] args) {

    /* Bucles */

    // 1. For

    for (int i = 0; i < 3; i++)
    {
        Console.Write(i);
    }

    // 2. Foreach

    // - Ejemplo 1:
    string[] animales = { "perro", "gato", "tigre" };

    foreach (string animal in animales)
    {
        Console.WriteLine(animal);
    }

    // - Ejemplo 2:

    void multiplicar(int primero, int segundo)
    {
        Console.WriteLine(primero * segundo);
    }

    int[] listaNumeros =  { 23, 45, 16, 37, 3, 99,22 };

    foreach (int numero in listaNumeros)
    {
        multiplicar(numero, 2);
    }

    // 3. While

    int entero = 100;
    int emergencia = 911;

    while (entero <= emergencia)
    {
        Console.WriteLine(entero);
        entero++; // Resultado: 100, 101, 102, 103, ... 911
    }

    // 4. Do While

    int valorInicial = 0;
    do
    {
        Console.Write(valorInicial);
        valorInicial++;
    } while (valorInicial < 5);
    // Output: 01234
  }
}
